types/lhr/audit-result.d.ts

```diff
@@ -4,10 +4,10 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {FormattedIcu} from './i18n';
-import AuditDetails from './audit-details';
+import {FormattedIcu} from './i18n.js';
+import AuditDetails from './audit-details.js';
 
-interface ScoreDisplayModes {
+export interface ScoreDisplayModes {
```

Comment:
Reason to export these? They don't appear to be used outside this file

Reply:
Might have been from an older version of this PR, I'll try removing.